packages/types/src/wallet.ts

```diff
@@ -8,8 +13,8 @@ export interface WalletCreate {
 // Stored in chrome.local.storage
 export interface WalletJson {
   label: string;
-  id: string;
-  fullViewingKey: string;
+  id: Stringified<WalletId>;
+  fullViewingKey: Stringified<FullViewingKey>;
   custody: { encryptedSeedPhrase: BoxJson };
 }
```

Comment:
`WalletJson` is stored in **chrome.local.storage** and `fullViewingKey` and `id` fields must be strings because we can only store strings in local storage. 
If before it was just bech32 strings, now it should be `Stringified<FullViewingKey>` and `Stringified<WalletId>`